flake.nix

```diff
@@ -114,15 +123,64 @@
         golden-gate-runtime = craneLib.buildPackage (common-wasm-attrs // {
           installPhase = ''
             mkdir --parents $out/lib
-            cp ./target/wasm32-unknown-unknown/release/wbuild/golden-gate-runtime/golden_gate_runtime.compact.compressed.wasm $out/lib
+            cp ./target/wasm32-unknown-unknown/release/wbuild/${common-wasm-attrs.pname}/golden_gate_runtime.compact.compressed.wasm $out/lib
           '';
           src = rust-src;
           cargoArtifacts = common-wasm-deps;
         });
+
+        golden-gate-node = craneLib.buildPackage (common-native-release-attrs // {
+          src = rust-src;
+          cargoArtifacts = common-native-release-deps;
+          nativeBuildInputs = common-native-release-attrs.nativeBuildInputs ++ [ pkgs.git ]; # parity does some git hacks in build.rs 
+        });
+
+
+        # really need to run as some points:
+        # - light client emulator (ideal for contracts)
+        # - multi node local fast (fast druation low security)
+        # - multi local slow (duration and security as in prod)
+        # - here can apply above to remote with something if needed (terranix/terraform-ng works)
+        # for each 
+        # - either start from genesis
+        # - of from fork (remote prod data)
+        # all with - archieval and logging enabled
+
+        single-fast = pkgs.writeShellApplication rec {
+          name = "single-fast";
+          text = ''
+            ${pkgs.lib.meta.getExe golden-gate-node} --dev  
+          '';
+        };
+
+        # we do not use existing Dotsama tools as they target relay + parachains
+        # here we can evolve into generating arion/systemd/podman/k8s output (what ever will fit) easy 
+        multi-fast = pkgs.writeShellApplication rec {
```

Comment:
Can you tell me how to run it? Just curious

Reply:
nix run .#multi-fast for curious ones.

Though, I think it's better to specify ports:
9001, 9002, 9003. 

cause sometimes you need to be connected to specific node